.ci/release-pipeline.yml

```diff
@@ -14,9 +14,14 @@ jobs:
   steps:
   - script: |
       echo "##vso[task.setvariable variable=version]$(git tag --points-at HEAD)"
-  - template: templates/publish.yml
-    parameters:
-      version: $(version)
+  - bash: ./gradlew --no-daemon -Pversion=$GRADLE_VERSION publish
+    env:
+      GRADLE_VERSION: $(version)
       ${{ if not(endsWith(variables['version'], '.RELEASE')) }}:
-        repoId: libs-milestone-local
-      # TODO support release publishing
+        GRADLE_PUBLISH_REPO_URL: https://repo.spring.io/libs-milestone-local/
+        GRADLE_PUBLISH_MAVEN_USER: $(artifactory.username)
+        GRADLE_PUBLISH_MAVEN_PASSWORD: $(artifactory.password)
+      ${{ if endsWith(variables['version'], '.RELEASE') }}:
+        GRADLE_PUBLISH_REPO_URL: https://api.bintray.com/maven/spring/jars/BlockHound/
```

Comment:
This isn't right, I think. for instance pool is released into `bintray.com/spring/jars/io.projectreactor.addons/`, so here it should be `io.projectreactor.tools` instead of `BlockHound`? Not sure how the publish plugin is configured, but the folder structure root should be at least modified

Reply:
one can create multiple "named packages" on Bintray. They don't have to follow the Maven package structure and serve as "bundles".

`io.projectreactor.tools` would be hard to deal with as soon as we get another project under it, because one release will mess with another (especially if you use auto publishing, and we may use it in future)

AFAIK it is recommended to use separate "named packages" to separate different projects under the same Bintray repo

Reply:
to rephrase:
`https://api.bintray.com/maven/spring/jars/BlockHound/` means that it will be published to a bundle on Bintray named "BlockHound", the package will remain `io.projectreactor.tools` tho

Reply:
Not to mention that BlockHound is **hosted** under `io.projectreactor` package, but is not actually a Reactor-specific project

Reply:
ok, BlockHound is its own thing. I know the bintray bundle is not the same as the maven groupId. Just seems to me it would keep things consistent.

Reply:
theoretically we could use something like `io.projectreactor.tools:BlockHound` (I've seen such notation in a few packages in our Bintray), although I'm not sure how Maven URL will be constructed in this case.